core/src/main/java/org/apache/iceberg/util/PropertyUtil.java

```diff
@@ -57,7 +57,7 @@ public static int propertyAsInt(Map<String, String> properties,
   }
 
   public static long propertyAsLong(Map<String, String> properties,
-                                    String property, long defaultValue) {
+                                    String property, Long defaultValue) {
```

Comment:
Not a huge fan of changing signature , even this minor way.

Would a better option be just to change the consuming code to expect a different non-null default value?  (like -1)

Reply:
Thanks for your opinion，have updated.